environment-sample

```diff
@@ -2,4 +2,5 @@ export LOGS_DIR=changeme
 export SLACK_LOGS_CHANNEL=changeme
 export SLACKBOT_API_TOKEN=changeme
 export GITHUB_WEBHOOK_SECRET=changeme
-export GITHUB_WEBHOOK_PORT=changeme
+export EBMBOT_WEBHOOK_SECRET=changeme
+export WEBHOOK_ORIGIN=http(s)://changeme:1234
```

Comment:
Can we add comments for these?

Reply:
I'll add a comment to say that they're commented in settings.py